package.json

```diff
@@ -26,6 +25,7 @@
   },
   "dependencies": {
     "lodash.clonedeep": "^4.5.0",
-    "ts-mixer": "^5.1.0"
+    "assemblyscript": "^0.18.30",
```

Comment:
While this library requires `assemblyscript` it is a peer dependency. I removed it a while back when asc versions were changing quickly and this library not keeping up meant a lot of warnings when users installed the library.  However, as you point in it is very important that the users of this library use the version of `asc` that it is made to handle.